features/built_in_matchers/predicates.feature

```diff
@@ -159,14 +159,14 @@ Feature: Predicate matchers
             end
           end
 
-          context 'with non-strict matchers (default)' do
-            let(:strict) { false }
+          context 'with strict matchers (default)' do
+            let(:strict) { true }
+            # deliberate failure
             it { is_expected.to have_strange_result }
           end
 
-          context 'with strict matchers' do
-            let(:strict) { true }
-            # deliberate failure
+          context 'with non-strict matchers' do
+            let(:strict) { false }
```

Comment:
Shouldn't this change the output below?

Reply:
No, I've just swapped those two examples so that the new `(default)` is on top.

Reply:
Yes but that inverts the output here, as it does above...

Reply:
L#176 should now be `to be truthy`

Reply:
No, the deliberate failure is for the strict option. As `has_strange_result?` returns 42, it passes with non-strict, and with it still fails with `return true`.

Reply:
Ah I see you've just swapped the two examples 🤦

Reply:
☕ 😄